core/src/main/java/sem/group15/bubblebobble/core/objects/Player.java

```diff
@@ -160,6 +160,13 @@ public void handleCollision(final GameObject other) {
                     logger.log("Player touched bubble");
                 }
             }
+            if (other instanceof Fruit) {
+                Fruit fruit = (Fruit) other;
+                if (fruit.getAliveTime() > 0.5) {
```

Comment:
The fruit spawns at the location of the enemy so without this the fruit gets picked up instantly after popping a bubble and you never see it or have to collect it.  <review